### DISCUSSION TITLE
What do I need to do if I want to customize the elements of the front-end page?

### DISCUSSION THREAD

**Discussion by Hourunze1997** [FreshRSS/FreshRSS#7094] [Q&A] [answered]
What do I need to do if I want to customize the elements of the front-end page?

**Comment by Frenzie** [comment]
What do you mean precisely? Under settings → display you can choose which icons to show above and under an article.

**Reply by Hourunze1997** [reply]
I want to customize the elements on the front-end page, such as logo, font, whatever, thank you for your answer

**Comment by Alkarex** [comment, marked as answer]
There are multiple options already, and several themes to chose from.
It is also possible to use the built-in userCSS extension to tune your CSS, or develop your own theme.
Examples: https://github.com/FreshRSS/FreshRSS/tree/edge/p/themes

**Reply by Hourunze1997** [reply]
thank you for your answer
